.github/workflows/api_inference_upload_pr_documentation.yml

```diff
@@ -0,0 +1,16 @@
+name: Upload Inference API PR Documentation
```

Comment:
https://github.com/huggingface/hub-docs/blob/main/.github/workflows/build_pr_documentation.yml for API-inference is missing

Reply:
thanks for spotting, it was in my untracked files ^^